src/components/List/index.jsx

```diff
@@ -9,13 +10,16 @@ export const Radio = props => {
 }
 
 export const List = props => {
-  const { className, bordered, ...rest } = props
+  const { className, border, paper, ...rest } = props
 
   return (
     <div
       className={cx(
         {
-          [styles['List--bordered']]: bordered
+          [styles['List--bordered']]: border === true,
+          [styles['List--papered']]: paper === true,
```

Comment:
I think we should set `papered` class for any truthy value. The user of the component can do something like

```jsx
<List papered={something && somethingElse}>
	{/* ... */}
</List>
```

Where `something` and `somethingElse` can be truthy value, so the result of the expression will be a truthy value too.

Reply:
I am not sure about that, I prefer to be precise about the types. What should I put in proptypes to support `papered={something && somethingElse}` ?

Reply:
Since any type is either truthy or falsy, there's nothing more precise than `PropTypes.any`. I understand your point of view, we can merge as is, that's not so important.